mteb/tasks/Clustering/fra/MLSUMClusteringP2P.py

```diff
@@ -38,6 +38,21 @@ class MLSUMClusteringP2P(AbsTaskClustering):
         avg_character_length=None,
     )
 
+    def load_data(self, **kwargs):
+        """
+        Load dataset from HuggingFace hub and convert it to the standard format.
+        """
+        if self.data_loaded:
+            return
+        self.dataset = datasets.load_dataset(
+            self.metadata.dataset["path"],
+            self.metadata.dataset["name"],
+            split=self.metadata.eval_splits[0],
```

Comment:
@KennethEnevoldsen In `AbsTask`, we pass `**self.metadata_dict["dataset"]` to the `load_dataset` function. Do we want to load all eval_splits instead? 

(I.e. rename the eval_split key to `split`, or add it when unpacking?)

Reply:
It could save us quite a bit of downloaded data. So it seems like a great idea (though I would probably keep it as `eval_splits`. This is a separate PR though, which I know will break some datasets (as the dataset_transform assume the whole dataset).